### PR TITLE
Multiple security patches in dependencies

### PR DESCRIPTION
**Pull request by snesm**
Updates to mitigate vulnerabilities in dependencies or sub-dependencies:

Critical org.thymeleaf:thymeleaf Sandbox Bypass
High net.minidev:json-smart Denial of Service (DoS)
Medium org.yaml:snakeyaml Arbitrary Code Execution
Medium com.squareup.okio:okio-jvm Denial of Service (DoS)
Low com.google.guava:guava Information Disclosure
Low com.google.guava:guava Creation of Temporary File in Directory with Insecure Permissions
Low org.jetbrains.kotlin:kotlin-stdlib Information Exposure

### CODE REVIEW

**Comment by snesm** [comment]
@LisaWellman @pbhallam @klwhaley @evbaron: any chance of getting these security fixes merged and released?

**Comment by klwhaley** [comment]
> @LisaWellman @pbhallam @klwhaley @evbaron: any chance of getting these security fixes merged and released?

Reviewing now!  Will work with @LisaWellman to get a release out as well.

File: build.gradle [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies {
 api 'org.apache.commons:commons-math3:3.6.1'
 
 // This dependency is used internally, and not exposed to consumers on their own compile classpath.
-implementation 'com.google.guava:guava:30.0'
+implementation 'com.google.guava:guava:32.0.0-android'
```
Comment (klwhaley) [inline build.gradle:28]: Wouldn't we want 32.0.0-jre not -android?

**Review by klwhaley — APPROVED** [review]
Thank you for the update

**Pull request merged**